README.md

```diff
@@ -60,27 +60,27 @@ pip install --user mal
 
 ### Manual Installation
 
-Clone this project and run inside it:
+Clone this project and inside it run:
 
 ```
 make install
 ```
 
-Probably we'll need have super-user permissions, but I'd recommend you
-to install inside of a virtualenv or use the `pip install --user' stuff.
+It is likely that `mal` will never need super-user permissions, but we recommend its
```

Comment:
Actually this is a little misleading, because `make install` needs super-user permissions if we call inside of a virtualenv. 
Probably the most correct way to fix all this thing is just point the manual installation as `pip install --user .` which will build the package and install in the userspace /home/$USER.